### PR TITLE
Fix a LG bug with a corner case in multiline scenario

### PR DESCRIPTION
**Pull request by Danieladu**
Issue link: https://github.com/microsoft/BotFramework-Composer/issues/4262

## Description
Originally, LG would break for the below  case:
```
# template
- ```
```
The cause is that LG does not allow empty multiline-text in multiline area before. 
Make the multiline-text could be empty to fix this issue.

## Specific Changes
- Change the parser file of LG to allow the empty multiline text
- Add a test to verify this change

### CODE REVIEW

File: libraries/botbuilder-lg/tests/lgDiagnostic.test.js [hunk 1]
```diff
@@ -187,6 +188,14 @@ describe(`LGExceptionTest`, function() {
         assert.strictEqual(diagnostics[0].message.includes(TemplateErrors.noEndingInMultiline), true);
     });
 
+    it(`TestMultiLineTemplate`, function() {
+        var diagnostics = preloaded.MultiLineTemplate;
```
Comment (joshgummersall) [inline libraries/botbuilder-lg/tests/lgDiagnostic.test.js:192]: `const`

File: libraries/botbuilder-lg/tests/lgDiagnostic.test.js [hunk 2]
```diff
@@ -187,6 +188,14 @@ describe(`LGExceptionTest`, function() {
...
+
+        assert.strictEqual(1, diagnostics.length);
```
Comment (joshgummersall) [inline libraries/botbuilder-lg/tests/lgDiagnostic.test.js:194]: These params are backward; `actual` comes first and `expected` is second.

**Review by joshgummersall — CHANGES_REQUESTED** [review]

File: libraries/botbuilder-lg/tests/lgDiagnostic.test.js (hunk 2 above)
Comment (Danieladu) [reply libraries/botbuilder-lg/tests/lgDiagnostic.test.js]: Yes. I fixed some of them. I found there are many places that should change. I think I could file an issue and fix them all at one time besides this PR. Is that ok?

Comment (joshgummersall) [reply libraries/botbuilder-lg/tests/lgDiagnostic.test.js]: Yeah, great idea!

**Review by joshgummersall — APPROVED** [review]

**Pull request merged**